### ISSUE TITLE
KV Host HA/instance HA with shared mountpoint

### ISSUE THREAD

**Issue by nomisunrider** [apache/cloudstack#9750]
##### ISSUE TYPE
 * Enhancement Request

##### COMPONENT NAME
KV Host HA/instance HA

##### CLOUDSTACK VERSION
N/A

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
KVM

##### SUMMARY
Host HA support for shared mountpoint.

Many VMware users looking at Cloudstack have FC/iSCSI block storage devices for their VMs.
Has there been any investigation/are there any plans to add KVM HA support for shared mountpoint using clustered filesystems such as OCFS2/GFS2?

**Comment by nomisunrider** [comment]
That workaround would be for enabling HA hosts, but would not give any HA instances, correct?  As I read the documentation I understand the instances must be hosted on NFS storage, so not sure what the workaround would really achieve.

Thank you

**Comment by nomisunrider** [comment]
Yes, I am looking if VM HA can be supported on a shared mount point in the event of a host failure.

Following your two steps would would result in VM HA in the event a host goes offline?  The VMs would power back up on another host?

Thank you

**Comment by sureshanaparti** [comment]
Addressed in #12773
